pxf/pxf-api/src/main/java/org/apache/hawq/pxf/api/examples/DummyFragmenter.java

```diff
@@ -0,0 +1,39 @@
+package org.apache.hawq.pxf.api.examples;
+
+import org.apache.hawq.pxf.api.Fragmenter;
+import org.apache.hawq.pxf.api.Fragment;
+import org.apache.hawq.pxf.api.utilities.InputData;
+import java.util.List;
+
+/**
+ * Fragmenter class for text data on local filesystem.
```

Comment:
According to logic, it's not really a local filesystem fragmenter, so please update comments.